app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def index
+    render '/posts/index'
+  end
+
+  def show
+    render '/posts/show'
+  end
+end
```

Comment:
-You have done well so far, but according to the project requirements, you are required to setup corresponding views for each of your controller actions. please fix this 👍